nx-X11/programs/Xserver/Xext/saver.c

```diff
@@ -1515,7 +1517,6 @@ SProcScreenSaverSuspend (ClientPtr client)
 
     swaps(&stuff->length);
     REQUEST_SIZE_MATCH(xScreenSaverSuspendReq);
-    swapl(&stuff->suspend);
```

Comment:
NAK.

X.Org removed this, but as far as I can tell this is actually an oversight/a bug. I'll poke X.Org to find out why they thought that might be a good idea.